client/my-sites/plans-features-main/index.jsx

```diff
@@ -213,11 +213,10 @@ class PlansFeaturesMain extends Component {
 				/>
 
 				<FAQItem
-					question={ translate( 'Can I upload my own plugins?' ) }
+					question={ translate( 'Can I install plugins?' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Install Plugins' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).